packages/nx/src/utils/target-project-locator.ts

```diff
@@ -135,22 +137,17 @@ export class TargetProjectLocator {
     return;
   }
 
-  private findProject(importExpr: string): string | undefined {
-    if (this.projectResolutionCache.has(importExpr)) {
```

Comment:
Do you think its worth keeping the cache? Or I guess require.resolve may have some caching, I know regular require does.

Reply:
`require` has its own cache:
https://nodejs.org/api/modules.html#modules_caching

But it should also be rare for it to get down there.